### ISSUE TITLE
A11Y issue - [Auro-Drawer] Hyperlinks are not focusable

### ISSUE THREAD

**Issue by megrenn** [AlaskaAirlines/auro-hyperlink#273]
### What browser client(s) are you seeing the issue on?

- [ ] Firefox
- [ ] Chrome
- [ ] Safari
- [ ] Microsoft Edge

### What devices platform(s) are you seeing the issue on?

- [ ] Desktop
- [ ] Mobile

### Steps to reproduce

Bringing in ticket from MBX ADO: https://dev.azure.com/itsals/E_Sell_Book/_workitems/edit/1261315

When tabbing through a full screen modal like SSRs, auro-hyperlinks are not focusable/accessible
Open SSRs
Tab through fields
Notice the hyperlinks at the bottom of the page are not highlighted or accessible via space/enter key

RESULT:
Focus stays on the last focusable element (either the final checkbox or the back/continue buttons) and the user can't interact with the hyperlinks via keyboard

EXPECTED RESULT:
Focus should hit each hyperlink as well

### Reproducible on the Auro doc site

- [x] I am able to reproduce this issue on the [Auro doc site](https://auro.alaskaair.com).

**Comment by megrenn** [comment]
Due by Monday 5/19

**Comment by blackfalcon** [comment]
Note: I don't believe that this is a hyperlink issue. After reviewing the dialog and drawer in the current Auro site I can say this. 

1. The hyperlink is properly accessible when in standard use 
2. The hyperlink is visually and keyboard accessible in the auro-dialog UI
3. The hyperlink is NOT keyboard accessible in the auro-drawer
4. Both the auro-drawer and auro-hyperlink do not give visual feedback when tabbing to the `x` close icon
5. The update to auro-drawer regarding keyboard `close` actions (`x`, `esc`, or `close` button) and returning the tab location back to the original button is not applied to the auro-dialog. 

It should be noted that the hyperlink is keyboard accessible when using the specific VoiceOver keys when using the screen reader.

**Comment by megrenn** [comment]
fixed as part of these two tickets: https://github.com/AlaskaAirlines/auro-hyperlink/issues/271
https://github.com/AlaskaAirlines/auro-formkit/issues/539
